tests/messages/test_extract.py

```diff
@@ -131,6 +140,17 @@ def test_multiline(self):
         self.assertEqual([(1, 'ngettext', (u'pylon', u'pylons', None), []),
                           (3, 'ngettext', (u'elvis', u'elvises', None), [])],
                          messages)
+        buf = BytesIO(b"""\
```

Comment:
These tests are nice enough, but can you move them to another function, `def test_npgettext(self):` for instance? :)